doc/23_Installation_and_Upgrade/07_Updating_Pimcore/12_V10_to_V11.md

```diff
@@ -2,6 +2,15 @@
 
 ## IMPORTANT CHANGES TO DO PRIOR THE UPDATE!
 
+### Migrations
+Make sure that migrations are executed.
+Pimcore does only run Pimcore core migrations after `composer` update automatically. If you'd like to run all available migrations including bundles and your app-specific migrations, just include the following part in your `composer.json`.
```

Comment:
> Pimcore does only run Pimcore core migrations after `composer` update automatically.

That's not true anymore, or is it?
See #12759 and pimcore/skeleton#113

Reply:
Yes sorry my mistake. I changed it and then copied everything from the docs again! 
Thank you! Just changed it.